extensions/wikia/CreateNewWiki/maintenance/importStarter.php

```diff
@@ -158,7 +158,14 @@ public function execute() {
 		/* @var Language $wgContLang */
 		global $wgContLang, $wgCityId, $wgDBname;
 		$language = $wgContLang->getCode();
-		$starter = Wikia\CreateNewWiki\Starters::getStarterByLanguage( $language );
+		$wikiVertical = (int) getopt( '', [ 'vertical:' ] )['vertical'];
+
+		// TODO Clean up in XW-2393
+		if ( $wikiVertical === \WikiFactoryHub::VERTICAL_ID_TV && $language === 'en' ) {
+			$starter = Wikia\CreateNewWiki\Starters::STARTER_EN_TV;
+		} else {
+			$starter = Wikia\CreateNewWiki\Starters::getStarterByLanguage( $language );
```

Comment:
Let's move the logic from here to `getStarterByLanguage` method (that should be removed to `getStarterByLanguageAndVertical`) to avoid mixing the responsibility of the code.